docs/UserGuide.md

```diff
@@ -137,7 +137,7 @@ This section describes the requirements of the specified data fields.
 
 ### Name
 * Names must only contain letters and spaces, and cannot be blank.
-* Names have a maximum allowed length of 54 characters in line with Singaporean law.
+* Names must contain at least one letter.
```

Comment:
I think this is unnecessary, since the previous line already mentions they cannot be blank.

Reply:
I think he wants to account for the case where the user enters only whitespaces, thinking that's not considered "blank"!

(Side note: I realised that we only specify the cannot-be-blank condition for names, but all the other fields except variant also can't be blank. Maybe we should move the cannot-be-blank condition to somewhere else and state that it applies to the other input fields as well?)

Reply:
Thanks for the clarification!

Would "must contain at least one letter" account for "cannot be blank"? Then, we'll only need the "must contain at least one letter" condition.

For the second point, I think the cannot-be-blank condition can be specified in the UG's "Notes about the command format" section. Something like:

* Input parameters must contain at least one non-whitespace character (i.e. blank inputs are not accepted)

Reply:
```suggestion
* Names must contain at least one letter.
```

Reply:
can can seems good